plugins/graphql-java-codegen-maven-plugin/pom.xml

```diff
@@ -0,0 +1,247 @@
+    <packaging>maven-plugin</packaging>
+    <version>1.5.0-SNAPSHOT</version>
+
+    <name>graphql-codegen-maven-plugin</name>
+    <description>Provides a goal for generating Java code based on GraphQL schema</description>
+    <url>https://github.com/kobylynskyi/graphql-java-codegen-maven-plugin</url>
+
+    <licenses>
+        <license>
+            <name>MIT License</name>
+            <url>https://github.com/kobylynskyi/graphql-java-codegen-maven-plugin/blob/master/LICENSE.md</url>
+        </license>
+    </licenses>
+
+    <developers>
+        <developer>
+            <id>kobylynskyi</id>
+            <name>Bogdan Kobylynskyi</name>
+            <url>https://github.com/kobylynskyi</url>
+            <email>92bogdan@gmail.com</email>
+        </developer>
+    </developers>
+
+    <issueManagement>
+        <system>GitHub</system>
+        <url>https://github.com/kobylynskyi/graphql-java-codegen/issues</url>
+    </issueManagement>
+
+    <scm>
+        <connection>scm:git:git://github.com/kobylynskyi/graphql-java-codegen-maven-plugin.git</connection>
+        <developerConnection>scm:git:git@github.com:kobylynskyi/graphql-java-codegen-maven-plugin.git
+        </developerConnection>
+        <url>https://github.com/kobylynskyi/graphql-java-codegen-maven-plugin</url>
+        <tag>v1.4.2</tag>
+    </scm>
+
+    <distributionManagement>
+        <snapshotRepository>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/content/repositories/snapshots</url>
+        </snapshotRepository>
+        <repository>
+            <id>ossrh</id>
+            <url>https://oss.sonatype.org/service/local/staging/deploy/maven2/</url>
+        </repository>
+    </distributionManagement>
+
+    <ciManagement>
+        <system>Travis</system>
+        <url>https://travis-ci.com/kobylynskyi/graphql-java-codegen-maven-plugin</url>
+    </ciManagement>
+
+    <properties>
+        <version.maven-plugin-api>3.6.2</version.maven-plugin-api>
+        <version.maven-core>3.6.2</version.maven-core>
+        <version.maven-plugin-annotations>3.6.0</version.maven-plugin-annotations>
+        <version.maven-plugin-plugin>3.6.0</version.maven-plugin-plugin>
+        <version.maven-compiler-plugin>3.8.1</version.maven-compiler-plugin>
+        <version.maven-resources-plugin>3.1.0</version.maven-resources-plugin>
+        <version.maven-source-plugin>3.1.0</version.maven-source-plugin>
+        <version.maven-javadoc-plugin>3.1.1</version.maven-javadoc-plugin>
+        <version.maven-deploy-plugin>3.0.0-M1</version.maven-deploy-plugin>
+        <version.nexus-staging-maven-plugin>1.6.8</version.nexus-staging-maven-plugin>
+        <version.maven-release-plugin>2.5.3</version.maven-release-plugin>
+        <version.maven-scm-provider-gitexe>1.11.2</version.maven-scm-provider-gitexe>
+        <version.maven-gpg-plugin>1.6</version.maven-gpg-plugin>
+
+        <version.graphql-java-codegen>1.5.0-SNAPSHOT</version.graphql-java-codegen>
```

Comment:
Same here, shouldn't this use the current version variable instead of a hardcoded version that happens to be equal to the current one?

Reply:
Fixed.